packages/mdc-select/_mixins.scss

```diff
@@ -98,12 +106,11 @@
 }
 
 @mixin mdc-select-focused-bottom-line-color_($color) {
-  .mdc-select__surface:focus .mdc-select__bottom-line,
-  .mdc-select__surface:focus ~ .mdc-select__bottom-line {
+  @include mdc-select-focused-bottom-line {
     @include mdc-theme-prop(background-color, $color);
   }
 
-  .mdc-select--open .mdc-select__bottom-line {
+  &.mdc-select--open .mdc-select__bottom-line {
```

Comment:
why do you need this CSS selector? why is mdc-select-focused-bottom-line not enough?

Reply:
`mdc-select-focused-bottom-line` only covers the "focused" state. The component is no longer focused when the menu is open, the menu is focused. When the menu closes, the component regains the "focused" state.

Reply:
ahh...okay add a comment above this CSS selector